src/analyses/ai.h

```diff
@@ -11,10 +11,11 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <map>
 #include <iosfwd>
-#include <sstream>
 
 #include <util/json.h>
 #include <util/xml.h>
+#include <util/expr.h>
+#include <util/simplify_expr.h>
```

Comment:
Why is this include required here? (I'd also debate the expr.h as it could be replaced by a forward declaration `class exprt;` but that's minor.)